rt/bindings/corba/src/main/java/org/apache/cxf/binding/corba/utils/CorbaAnyHelper.java

```diff
@@ -273,257 +267,19 @@ public static void extractPrimitiveFromAny(Any a, CorbaPrimitiveHandler primitiv
     }
 
     private static Any createFixedAny(ORB orb, Any any) {
-        createFixedAnyConstructor();
+        if (fixedAnyConstructor == null) {
```

Comment:
Don't like that, but we probably need `synchronized` block here, the original static method `createFixedAnyConstructor` was declared `static synchronized` probably to protected against concurrent generation.